facia-end-to-end/conf/routes

```diff
@@ -1,94 +1,97 @@
 # Routes
 
 #CUSTOM (This Project)
-GET        /configagent         controllers.FaciaEndToEndController.configAgentContents
+GET         /configagent                                                    controllers.FaciaEndToEndController.configAgentContents
 
 #FACIA-PRESS
-GET        /_healthcheck        conf.HealthCheck.healthcheck()
-GET        /debug/config        controllers.Application.showCurrentConfig
+GET         /_healthcheck                                                   conf.HealthCheck.healthcheck()
+GET         /debug/config                                                   controllers.Application.showCurrentConfig
 
-GET        /pressed/fronts      controllers.Application.generateFrontJson()
-GET        /pressed/live/*id      controllers.Application.generateLivePressedFor(id)
+GET         /pressed/fronts                                                 controllers.Application.generateFrontJson()
+GET         /pressed/live/*id                                               controllers.Application.generateLivePressedFor(id)
 
 #FACIA-TOOL
 # authentication endpoints
-GET           /authError                 controllers.PandaAuthController.authError(message:String)
-GET           /oauthCallback             controllers.PandaAuthController.oauthCallback
-GET           /logout                    controllers.PandaAuthController.logout
-GET           /login/status              controllers.PandaAuthController.status
+GET         /authError                                                      controllers.PandaAuthController.authError(message:String)
+GET         /oauthCallback                                                  controllers.PandaAuthController.oauthCallback
+GET         /logout                                                         controllers.PandaAuthController.logout
+GET         /login/status                                                   controllers.PandaAuthController.status
+
+# An sbt multi-project needs to route to internal web jars, kept in lib
+GET         /assets/facia-tool/*file                                        controllers.faciatool.UncachedAssets.at(file, relativePath="lib/facia-tool/")
 
 # static files
-GET           /humans.txt                controllers.Assets.at(path="/public", file="humans.txt")
-GET           /assets/*path              dev.DevAssetsController.at(path)
+GET         /humans.txt                                                     controllers.Assets.at(path="/public", file="humans.txt")
+GET         /assets/*file                                                   controllers.Assets.at(path="/public", file)
```

Comment:
Thanks, I'll make sure it's using the DevAssetsController. That won't affect the behaviour of facia-tool in dev